tests/integration/config/test_views_suggesters.py

```diff
@@ -64,7 +65,6 @@ def test_literature_suggesters_empty_result(api_client, db, create_record):
     assert result == expected
 
 
-@pytest.mark.xfail(reason="author_suggest field is not populated when indexing authors")
 def test_author_suggesters(api_client, db, create_record):
```

Comment:
needs the ``es_clear`` fixture and drop the ``from invenio_search import current_search_client as es``